### PR TITLE
Add CDAP-UI Dependencies to license files

### PR DESCRIPTION
**Pull request by awholegunch**
As 3.0.1 has now been merged back to develop, this PR now includes just the 3.1.0 versions of the PDFs. However, there are improvements to the Python script requested. I will make them in this PR.

Staged at: 

- [Trademarks, Licenses, and Dependencies](http://docs-staging.cask.co/cdap/3.1.0-SNAPSHOT-feature-3.1_doc_ui_licenses/en/reference-manual/licenses/index.html)
- [Cask Data Application Platform UI Dependencies](http://docs-staging.cask.co/cdap/3.1.0-SNAPSHOT-feature-3.1_doc_ui_licenses/en/reference-manual/licenses/cdap-ui-dependencies.html)

Fix for https://issues.cask.co/browse/CDAP-2582

### CODE REVIEW

File: cdap-docs/tools/licenses/licenses.py [hunk 1]
```diff
@@ -28,20 +28,38 @@
 import subprocess
 import sys
 
-VERSION = "0.0.9"
+VERSION = "0.0.9.1"
```
Comment (sreevatsanraman) [inline cdap-docs/tools/licenses/licenses.py]: why are we changing versions the format?

Comment (awholegunch) [inline cdap-docs/tools/licenses/licenses.py]: My mistake. Fixed as "0.1.0".

**Comment by duddun** [comment]
LGTM (Note: some of the defs can use comments)

**Comment by awholegunch** [comment]
If #2948 gets merged and the pulled to develop, then this PR should not be merged but closed instead.

**Comment by awholegunch** [comment]
I'll create a new PR and update the script with comments.

**Pull request closed without merging**